Foundation/Host.swift

```diff
@@ -33,6 +33,18 @@ open class Host: NSObject {
     }
     
     static internal func currentHostName() -> String {
+#if os(Windows)
+      var dwLength: DWORD = 0
+      GetComputerNameExA(ComputerNameDnsHostname, nil, &dwLength)
+
+      let hostname: UnsafeMutablePointer<Int8>? =
```

Comment:
Does `hostname` need to be an optional?

Reply:
hmm, I suppose not anymore, since it is being assigned a value immediately.